data/sql/updates/pending_db_world/rev_1562779174835710547.sql

```diff
@@ -0,0 +1,157 @@
+VALUES
+(50004, 0, 0, 0, 0, 0, 17881, 0, 0, 0, 'Adyen Trigger', NULL, NULL, 0, 1, 1, 0, 1806, 1, 1, 1.14286, 1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 0, 3, 1, 1, 1, 1, 0, 0, 1, 0, 128, 'deathblow_to_the_legion_trigger', 0);
+DELETE FROM `creature` WHERE `id` = 50004;
+INSERT INTO `creature` (`id`, `map`, `modelid`, `position_x`, `position_y`, `position_z`, `orientation`) VALUES ('50004', '530', '17881', '4802.629883', '3775.419922', '210.529999', '5.490000');
+
+-- Socrethar 'socrethar'
+UPDATE `creature_template` SET `ScriptName` = 'socrethar', `faction` = 90, `flags_extra` = 2097152 WHERE `entry` = 20132;
+
+-- Ishanah 'npc_ishanah'
+UPDATE `creature_template` SET `ScriptName` = 'npc_ishanah' WHERE `entry` = 18538;
+
+-- Orelis gossip text
+DELETE FROM `gossip_menu` WHERE `MenuID` = 61021 AND `TextID` = 10210;
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES (61021, 10210);
+DELETE FROM `gossip_menu_option` WHERE `MenuID` = 61021 AND `OptionID` = 0;
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionIcon`, `OptionText`, `BoxBroadcastTextID`) VALUES ('61021', '1', 'I''m ready, Adyen', 18591);
+
+-- Adyen waypath
+DELETE FROM `waypoint_data` WHERE `id` = 610210;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(610210, 0, 4814.52, 3767.18, 210.535, 5.72712, 0, 0, 0, 100, 0),
+(610210, 1, 4836.75, 3774.64, 207.521, 0.31875, 0, 0, 0, 100, 0),
+(610210, 2, 4849.92, 3780.49, 203.479, 0.464049, 0, 0, 0, 100, 0),
+(610210, 3, 4860.43, 3791.42, 199.593, 0.808459, 0, 0, 0, 100, 0),
+(610210, 4, 4865.64, 3799.13, 199.073, 0.981247, 0, 0, 0, 100, 0),
+(610210, 5, 4884.83, 3808.67, 198.972, 0.451103, 0, 0, 0, 100, 0),
+(610210, 6, 4889.64, 3811.75, 202.37, 0.568913, 0, 0, 0, 100, 0),
+(610210, 7, 4898.13, 3817.17, 208.018, 0.568913, 0, 0, 0, 100, 0),
+(610210, 8, 4907.47, 3823.15, 211.487, 0.568913, 0, 0, 0, 100, 0),
+(610210, 9, 4925.36, 3834.58, 211.494, 0.568913, 0, 0, 0, 100, 0);
+
+-- Karja waypath
+DELETE FROM `waypoint_data` WHERE `id` = 500020;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(500010, 0, 4808.67, 3776.1, 210.545, 0.185895, 0, 0, 0, 100, 0),
```

Comment:
(500020, 0, 4808.67, 3776.1, 210.545, 0.185895, 0, 0, 0, 100, 0),

Reply:
Yes, need `;` in end

Reply:
Good catch, thanks! I corrected this and pushed a new commit.